### PR TITLE
feat(web): manual stacking asset

### PR DESCRIPTION
**Pull request by alextran1502**
This PR adds manual asset stacking on the web - for feature compatibility with the mobile app

https://github.com/immich-app/immich/assets/27055614/50df91e2-443d-455e-b5e3-24596f5eb658

### CODE REVIEW

File: server/src/infra/repositories/asset.repository.ts [hunk 1]
```diff
@@ -540,11 +543,9 @@ export class AssetRepository implements IAssetRepository {
         .andWhere('person.id = :personId', { personId });
     }
 
-    // Hide stack children only in main timeline
-    // Uncomment after adding support for stacked assets in web client
-    // if (!isArchived && !isFavorite && !personId && !albumId && !isTrashed) {
-    //   builder = builder.andWhere('asset.stackParent IS NULL');
-    // }
+    if (!isArchived && !isFavorite && !personId && !albumId && !isTrashed) {
```
Comment (jrasm91) [inline server/src/infra/repositories/asset.repository.ts:546]: This looks crazy. Why don't we just pass an `withStacked` on the main timeline route, probably from the client.

**Review by jrasm91 — APPROVED** [review]
Looks mostly good. The web stuff is a bit messy. We need to change how we handle the detail viewer nav actions.

File: web/src/lib/stores/assets.store.ts [hunk 2]
```diff
@@ -135,7 +135,7 @@ export class AssetStore {
     this.emit(true);
   }, 10_000);
 
-  async init(viewport: Viewport) {
+  async init(viewport: Viewport, withStacked = false) {
```
Comment (shenlong-tanwen) [inline web/src/lib/stores/assets.store.ts:138]: This is not needed anymore now.

**Review by shenlong-tanwen — APPROVED** [review]
LGTM.

**Pull request merged**